web/views.py

```
@@ -314,6 +317,47 @@ def signup_view(request):
     )
 
 
+@login_required
+def delete_account(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `delete_account`

(ANN201)